README.md

```diff
@@ -19,6 +19,7 @@ You want to contribute 🌱? Please read the [Contribution Guidelines](CONTRIBUT
   * [Camunda 8](#camunda-8)
   * [Camunda 7](#camunda-7)
 * 📈 [Reporting Carbon Reduction via Camunda Optimize](#reporting-carbon-reduction-via-camunda-optimize)
+* [🏗️ Building Blocks](#building-blocks)
```

Comment:
Der Sprung zum Anker funktioniert nicht.

Reply:
I don't get it. Using the IntelliJ Markdown Preview the anchors are working ...

I'll have a closer look

Reply:
There was a problem with the emoji ...